test/configuration_test.rb

```diff
@@ -117,6 +117,10 @@
         @mock_store1.data['file'].must_equal(:content => 'content', :content_type => 'text/plain', :last_update_ts => @file.last_update_ts)
         @mock_store2.data['file'].must_be_nil
       end
+
+      it 'should not adds the errors into the file' do
```

Comment:
should this say `should not add any errors into the file`? (otherwise it reads as if there are errors but they shouldn't be added)

Reply:
Aha yeah that makes sense, I'll change that